modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml

```diff
@@ -169,11 +169,11 @@
 				</if>
 
 				<echo>A Gogo shell opened successfully.</echo>
-
-				<stop-one-java-process process.name="jar" />
 			</sequential>
 		</parallel>
 
+		<stop-one-java-process process.name="jar" />
```

Comment:
I moved stopping the jar process outside of the parallel block because the kill command returns Java Result: 143, which causes Poshi to throw an exception and fail.

However, stopping the jar process outside of the parallel block doesn't execute the rest of the script to clean up the processes. So I isolated the tests using this stop jar process in https://github.com/vicnate5/liferay-portal/pull/2200/commits/5d3816cb56557d7de3e7ed92324230a75e39effe due to bad cleanup. 

**Example:** 
[LocalFile.DatabaseUpgradeClient#CheckUpgradeClientGogoShell
](https://testray.liferay.com/reports/production/logs/2022-12/test-1-39/test-portal-acceptance-pullrequest(master)/2084/functional-upgrade-tomcat90-mysql57-jdk8/0/9/jenkins-console.txt.gz?authuser=0)